nannou/src/wgpu/render_pipeline_builder.rs

```diff
@@ -28,56 +28,46 @@ pub struct RenderPipelineBuilder<'a> {
     fs_mod: Option<&'a wgpu::ShaderModule>,
     vs_entry_point: &'a str,
     fs_entry_point: &'a str,
-    rasterization_state: Option<wgpu::RasterizationStateDescriptor>,
-    primitive_topology: wgpu::PrimitiveTopology,
-    color_state: Option<wgpu::ColorStateDescriptor>,
-    color_states: &'a [wgpu::ColorStateDescriptor],
-    depth_stencil_state: Option<wgpu::DepthStencilStateDescriptor>,
-    index_format: wgpu::IndexFormat,
-    vertex_buffers: Vec<wgpu::VertexBufferDescriptor<'static>>,
-    sample_count: u32,
-    sample_mask: u32,
-    alpha_to_coverage_enabled: bool,
+    primitive: wgpu::PrimitiveState,
+    color_state: Option<wgpu::ColorTargetState>,
+    color_states: &'a [wgpu::ColorTargetState],
+    depth_stencil: Option<wgpu::DepthStencilState>,
+    vertex_buffers: Vec<wgpu::VertexBufferLayout<'static>>,
+    multisample: wgpu::MultisampleState,
```

Comment:
Nice job figuring out how to update the `RenderPipeline` (not just these lines but throughout)!